### PR TITLE
Added Tips and Tricks in Wiki.md - Markdown Preview

### PR DESCRIPTION
**Pull request by MohitGupta121**
Added Tips and Tricks in Wiki.md - Wiki Markdown Preview in Visual Studio Code

### CODE REVIEW

**Comment by MohitGupta121** [comment]
@gp201 PTAL, I make PR for preview as we discuss in the meeting.

**Review by MohitGupta121 — COMMENTED** [review]
@gp201 PTAL, I make PR for preview as we discuss in the meeting.

File: Wiki.md [hunk 1]
```diff
@@ -177,3 +177,12 @@ If the deployment of changes to the wiki following the merging of a pull request
 1. Navigate to the Oppia wiki repository's Actions tab.
 2. Select the "Deploy to wiki" workflow.
 3. Refer to these [instructions](https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow?tool=webui#running-a-workflow) to manually execute the workflow.
+
+## Tips and Try
```
Comment (gp201) [inline Wiki.md]:
```suggestion
## Tips and Tricks
```

Update the PR title also

File: Wiki.md [hunk 2]
```diff
@@ -177,3 +177,12 @@ If the deployment of changes to the wiki following the merging of a pull request
...
+
+* ### How can I see markdown (wiki) preview in Visual Studio Code?
+
+Right-click on the editor Tab and select **Open Preview** (Windows/Linux: `Ctrl+Shift+V`, Mac: `Command+Shift+V`) or use the **Command Palette** Windows/Linux: `Ctrl+Shift+V`, Mac: `Command+Shift+V` to run the **Markdown: Open Preview to the Side** command (Windows/Linux: `Ctrl+K V`, Mac: `Command+K V`).
```
Comment (gp201) [inline Wiki.md]: Is it possible to use the `<kbd>` tag for keys?

File: Wiki.md [hunk 3]
```diff
@@ -177,3 +177,12 @@ If the deployment of changes to the wiki following the merging of a pull request
...
 3. Refer to these [instructions](https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow?tool=webui#running-a-workflow) to manually execute the workflow.
+
+## Tips and Try
+
+* ### How can I see markdown (wiki) preview in Visual Studio Code?
+
+Right-click on the editor Tab and select **Open Preview** (Windows/Linux: `Ctrl+Shift+V`, Mac: `Command+Shift+V`) or use the **Command Palette** Windows/Linux: `Ctrl+Shift+V`, Mac: `Command+Shift+V` to run the **Markdown: Open Preview to the Side** command (Windows/Linux: `Ctrl+K V`, Mac: `Command+K V`). 
+
+#### Demo
+<img width="1074" alt="Screenshot wiki preview" src="https://user-images.githubusercontent.com/76530270/227735960-6dcd4453-4c5a-4eaf-b33b-48a68a2dbdfb.gif">
```
Comment (gp201) [inline Wiki.md:189]: The image is not rendering PTAL

**Review by gp201 — CHANGES_REQUESTED** [review]
Thanks, @MohitGupta121 ! I left some comments. PTAL

File: Wiki.md (hunk 3 above)
Comment (MohitGupta121) [reply Wiki.md:189]: It is .gif, so change I change it to png? (https://user-images.githubusercontent.com/76530270/227735960-6dcd4453-4c5a-4eaf-b33b-48a68a2dbdfb.gif)

Actually the .gif is rendering in Android studio wiki preview but not in VS Code. But its render in the github wiki.

Update: It's rendering everywhere please check, take some time to load I think.

Screenshot 2023-03-26 at 12 48 40 AM

File: Wiki.md (hunk 2 above)
Comment (MohitGupta121) [reply Wiki.md]: Yes it's possible, I'll add keboard symbols.

**Review by MohitGupta121 — COMMENTED** [review]
@gp201 PTAL, I added keyboard symbols but It's not looking good.
What you suggests?

**Review by MohitGupta121 — COMMENTED** [review]
@gp201 PTAL, updated as per review.

File: Wiki.md (hunk 2 above)
Comment (gp201) [reply Wiki.md]: @MohitGupta121 sry if I wasn't clear. I meant `<kbd>Shift</kbd>` for each key. This should render as <kbd>Shift</kbd>

**Comment by gp201** [comment]
@MohitGupta121 I left a comment [here](https://github.com/oppia/oppia-web-developer-docs/pull/148#discussion_r1154096633)

**Review by MohitGupta121 — COMMENTED** [review]
@gp201 PTAL, I updated the keyboard tags.
Thanks.

File: Wiki.md [hunk 4]
```diff
@@ -177,3 +177,12 @@ If the deployment of changes to the wiki following the merging of a pull request
 1. Navigate to the Oppia wiki repository's Actions tab.
 2. Select the "Deploy to wiki" workflow.
 3. Refer to these [instructions](https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow?tool=webui#running-a-workflow) to manually execute the workflow.
+
+## Tips and Tricks
+
```
Comment (gp201) [inline Wiki.md:184]: 1 final nit. Please add this to the table of contents on top of the page. Thanks!

**Review by gp201 — CHANGES_REQUESTED** [review]

File: Wiki.md (hunk 4 above)
Comment (MohitGupta121) [reply Wiki.md:184]: Done @gp201

**Comment by MohitGupta121** [comment]
@gp201 PTAL, done added Tips and Tricks in Table of Conent.
Thanks.

File: Wiki.md [hunk 5]
```diff
@@ -177,3 +178,12 @@ If the deployment of changes to the wiki following the merging of a pull request
...
+### How can I see markdown (wiki) preview in Visual Studio Code?
```
Comment (gp201) [inline Wiki.md:185]: you missed this one

File: Wiki.md [hunk 6]
```diff
@@ -177,3 +178,12 @@ If the deployment of changes to the wiki following the merging of a pull request
...
+
+Right-click on the editor Tab and select **Open Preview** (Windows/Linux: <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>V</kbd>, Mac: <kbd>Command</kbd>+<kbd>Shift</kbd>+<kbd>V</kbd>) or use the **Command Palette** Windows/Linux: <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>V</kbd>, Mac: <kbd>Command</kbd>+<kbd>Shift</kbd>+<kbd>V</kbd> to run the **Markdown: Open Preview to the Side** command (Windows/Linux: <kbd>Ctrl</kbd>+<kbd>K</kbd>+<kbd>V</kbd>, Mac: <kbd>Command</kbd>+<kbd>K</kbd>+<kbd>V</kbd>). 
+
+#### Demo
```
Comment (gp201) [inline Wiki.md]: Can we remove this heading?

**Review by gp201 — CHANGES_REQUESTED** [review]

File: Wiki.md (hunk 5 above)
Comment (MohitGupta121) [reply Wiki.md:185]: In Table of content?

File: Wiki.md (hunk 6 above)
Comment (MohitGupta121) [reply Wiki.md]: Okay sure.

File: Wiki.md (hunk 5 above)
Comment (MohitGupta121) [reply Wiki.md:185]: Done.

File: Wiki.md (hunk 6 above)
Comment (MohitGupta121) [reply Wiki.md]: Done.

**Review by MohitGupta121 — COMMENTED** [review]
@gp201 PTAL, Thanks.

File: Wiki.md (hunk 1 above)
Comment (MohitGupta121) [reply Wiki.md]: Done.

File: Wiki.md (hunk 2 above)
Comment (MohitGupta121) [reply Wiki.md]: Done.

**Review by gp201 — APPROVED** [review]
LGTM

**Comment by gp201** [comment]
Thanks @MohitGupta121

**Pull request merged**